csharp/ql/test/library-tests/ir/offbyone/null.cs

```diff
@@ -0,0 +1,5 @@
+class Null {
```

Comment:
I guess this file is to ensure the existence of the `NullType` in the database - until the IR type system gets merged.

Reply:
Exactly, without it we get a lot of fragmented blocks. Should be able to get rid of it once the IR type system is in place.